app/models/user.rb

```diff
@@ -125,7 +126,11 @@ def with_all_associations_joined
     def with_interest(interest)
       where(":interest = ANY(interested_in)", interest: interest)
     end
-  end
+
+    def non_updateable_params
```

Comment:
the `immutable_params` where created at first for future-proofing if there might be some changes with github_handle or more params added and also in order to call `User.immutable_params` in application_draft factory (+ user controller in update action - deleted in last commits). 
Application draft factory needed `User.immutable_params` to return symbol, but i wanted to test how to pass more symbols, and that's why i made an array and a splat.
As for failing tests - everything passes since	200f387 commit, now it does too.
As for `spec/controllers/users_controller_spec.rb:6` - "github_id" and avatar_info were in the test before I started to fix it. I guess it passed because i've had filtered Update_attributes in update action. But since i've deleted it -  I added `User.immutable_params.to_s` in the controller test to exclude github_handle. Hope it all makes some sense, if not please let me know how bad i've messed up this time ;p I will try to understad and fix it again :)